engine/src/main/java/org/hibernate/validator/internal/engine/ConstraintViolationImpl.java

```diff
@@ -327,9 +327,9 @@ public String toString() {
 	private int createHashCode() {
 		int result = interpolatedMessage != null ? interpolatedMessage.hashCode() : 0;
```

Comment:
Not strictly related, but it seems we could get rid of a few null checks here, for all the attributes we know will exist?